auth/management.py

```diff
@@ -0,0 +1,17 @@
+import requests
```

Comment:
Let's use `httpx` instead of `requests` for all web requests, mostly for consistency but also because `httpx` allows `async` requests